script/spec-runner.js

```diff
@@ -52,6 +62,14 @@ if (args.runners !== undefined) {
 }
 
 async function main () {
+  if (args.electronVersion) {
+    const versions = await ElectronVersions.create();
+    if (!versions.isVersion(args.electronVersion)) {
+      console.log(`${fail} ${args.electronVersion} is not a supported Electron version`);
```

Comment:
_nit_

This error message could be a little confusing since it's not really about supported / unsupported and there are valid use for running tests on an unsupported old release

```suggestion
      console.log(`${fail} '${args.electronVersion}' is not a recognized Electron version`);
```

Reply:
Good call out, you're right, the suggested message is more clear.